lib/vagrant-orchestrate/command/push.rb

```diff
@@ -13,6 +13,10 @@ def execute
           opts = OptionParser.new do |o|
             o.banner = "Usage: vagrant orchestrate push"
             o.separator ""
+
+            o.on("--reboot", "Reboot a managed server") do
```

Comment:
@jdaviscooke I think updating the description to say that it reboots after the provisioning steps occur would be helpful.